content/guides/getting-started/component-framework-configuration.md

```diff
@@ -51,17 +51,15 @@ const { devServer } = require('@cypress/react/plugins/react-scripts')
 {
   component: {
     devServer,
-    componentFolder: 'src',
-    testFiles: '**/*.test.{js,ts,jsx,tsx}'
+    specPattern: '**/*.cy.{js,jsx,ts,tsx}'
```

Comment:
It doesn't entirely make sense to recommend that the user specify a `specPattern` option with the default value. Also, the text before the example doesn't match the pattern. 

Perhaps change the text in the paragraph above to say something like this (and maybe link `specPattern` to the place in the configuration guide where it is explained):

> By default, Cypress looks for spec files anywhere in your project with an extension of either `.cy.js`, `.cy.jsx`, `.cy.ts`, or `.cy.tsx`. However, you can change this behavior with a custom `specPattern` value. In the following example, we've configured Cypress to look for spec files with those same extensions, but only in the `src` folder or any of its subdirectories.

And then change the `specPattern` to `'src/**/*.cy.{js,jsx,ts,tsx}'`

This comment applies to all 5 examples on this page.